traits/src/location.rs

```diff
@@ -78,12 +78,12 @@ impl Reserve for RelativeReserveProvider {
 }
 
 pub trait RelativeLocations {
-	fn sibling_parachain_general_key(para_id: u32, general_key: WeakBoundedVec<u8, ConstU32<32>>) -> MultiLocation;
+	fn sibling_parachain_general_key(para_id: u32, general_key: BoundedVec<u8, ConstU32<32>>) -> MultiLocation;
 }
 
 impl RelativeLocations for MultiLocation {
-	fn sibling_parachain_general_key(para_id: u32, general_key: WeakBoundedVec<u8, ConstU32<32>>) -> MultiLocation {
-		MultiLocation::new(1, X2(Parachain(para_id), GeneralKey(general_key)))
+	fn sibling_parachain_general_key(para_id: u32, general_key: BoundedVec<u8, ConstU32<32>>) -> MultiLocation {
+		return MultiLocation::new(1, X2(Parachain(para_id), general_key.as_bounded_slice().into()));
```

Comment:
this changes v2 to v3, which is fine, but can we have some tests to ensure the result from old code is same as the new code?

i.e. generate a v2 MultiLocation with old code, upgrade it to v3, assert it is the same

Reply:
The returned Multilocation differs as `GeneralKey` changed between versions